tyk-headless/values.yaml

```diff
@@ -168,7 +173,44 @@ gateway:
 # https://github.com/TykTechnologies/tyk-pump#configuration
 pump:
   # Determines whether or not the pump component should be installed.
-  enabled: false
+  enabled: true
+
+  # prometheusPump configures Tyk Pump to expose Prometheus metrics.
+  prometheusPump:
+    # enabled determines whether the Prometheus should be used by Tyk Pump.
+    enabled: true
+    # host represents the host without port, where Tyk Pump serve the metrics for Prometheus.
+    host: ""
+    # port represents the port where Tyk Pump serve the metrics for Prometheus.
+    port: 9090
+    # path represents the path to the Prometheus collection. For example /metrics.
+    path: /metrics
+    # customMetrics allows defining custom Prometheus metrics for Tyk Pump.
+    # It accepts a string that represents a JSON object. For instance,
+    #
+    # customMetrics: '[{"name":"tyk_http_requests_total","description":"Total of API requests","metric_type":"counter","labels":["response_code","api_name"]}]'
+    customMetrics: ""
+    # By default, Pump creates a PodMonitor Custom Resource on your cluster if you use Prometheus Pump. Set .prometheusOperator.enabled to false
+    # for disabling installation of PodMonitor resource.
+    prometheusOperator:
+      # enabled determines whether the Prometheus Operator is in use or not. By default,
+      # it is enabled.
+      # Tyk Pump can be monitored with PodMonitor Custom Resource of Prometheus Operator.
+      # If enabled, PodMonitor resource is created based on .Values.pump.prometheusPump.prometheusOperator.podMonitorSelector
+      # for Tyk Pump.
+      enabled: true
```

Comment:
If we disable it, then will prometheus pump work?

Reply:
ah, yes. it should be disabled. i forgot to switch off